### ISSUE TITLE
Bulk update and delete mutations in GraphQL API

### ISSUE THREAD

**Issue by marksurnin** [stargate/stargate#1223]
Would it be possible to add GraphQL mutation types for bulk updates and deletes with a `filter` condition? We have a number of use cases for updates and deletes with `WHERE` clauses. The current workaround is to do string manipulation to send multiple mutations in a single request.

**Comment by olim7t** [comment]
Yes, it should be pretty straightforward to add a `filter` argument to update and delete operations.

Note that it will be subject to the same rules as pure CQL queries:
* update: all partition keys and clustering columns must be restricted by EQ or IN relations
* delete: one clustering column can be restricted by other operators, but then no other clustering column after it can be restricted.

Could you give examples of the kind of queries you'd like to achieve?

**Comment by floriantraber** [comment]
This is a simplified example of a query we have: 

```
CREATE TABLE my_table (
field1 int,
field2 varchar,
field3 int,
PRIMARY KEY (field1, field2, field3)
)


DELETE
FROM my_table
WHERE field1 in ? AND field2 = ? AND field3 <= ?
```
